examples/jsm/loaders/NRRDLoader.js

```diff
@@ -1,351 +1,292 @@
-import {
-	FileLoader,
-	Loader,
-	Matrix4,
-	Vector3
-} from 'three';
-import * as fflate from '../libs/fflate.module.js';
-import { Volume } from '../misc/Volume.js';
+import { FileLoader, Loader, Matrix4, Vector3 } from "three";
```

Comment:
Can you please retain the code style and not format the entire file?

This makes it nearly impossible to review the actual changed lines of code.

Reply:
Yes no problem. I've already reset the code and made a new pull request. I will close this now. Thanks!